### DISCUSSION TITLE
How to use shortcuts with CDN runtime

### DISCUSSION THREAD

**Discussion by DavidSvi** [unocss/unocss#1502] [Q&A]
Hey there, I am using WordPress.

I followed the instructions here: https://github.com/unocss/unocss/tree/main/packages/runtime

I loaded the JS library code
<script src="https://cdn.jsdelivr.net/npm/@unocss/runtime"></script>

I loaded the CSS
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@unocss/reset/tailwind.min.css">

and then I added this code to setup shortcut but it is not working....

```
window.__unocss = {
  rules: [
    // custom rules...
  ],
  shortcuts: [
      {
  // shortcuts to multiple utilities
  'tab': 'py-2 px-4 font-semibold rounded-lg shadow-md',
  'btn-green': 'text-white bg-green-500 hover:bg-green-700',
  // single utility alias
  'red': 'text-red-100'
}
],
  presets: [
    // custom presets...
  ],
  // ...
}
```

I am not really sure what I should do to make it work. Appreciate any help.

Thank you

**Comment by DavidSvi** [comment]
I have also noticed that not all tailwind CSS works.

**Comment by MarioC3** [comment]
You need to add this part before you load the library:

```

window.__unocss = {
	// rules, shortcuts, presets, etc.
}
```

 So, something like this:

```
<script>
window.__unocss = {
	// rules, shortcuts, presets, etc.
}
</script>
<script src="https://cdn.jsdelivr.net/npm/@unocss/runtime"></script>
```
